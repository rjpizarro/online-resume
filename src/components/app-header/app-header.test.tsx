import React from 'react'
import renderer from 'react-test-renderer'

import AppHeader from './index'

describe("AppHeader", () => {
  it("renders normally", () => {
    const tree = renderer
      .create(<AppHeader siteTitle="Test site title" />)
      .toJSON()

    expect(tree).toMatchSnapshot()
  })

  it("renders without site title when the siteTitle prop is undefined", () => {
    const tree = renderer
      .create(<AppHeader />)
      .toJSON()

    expect(tree).toMatchSnapshot()
  })
})