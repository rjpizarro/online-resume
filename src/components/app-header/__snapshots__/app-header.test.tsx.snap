// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppHeader renders normally 1`] = `
<header
  style={
    Object {
      "background": "rebeccapurple",
      "marginBottom": "1.45rem",
    }
  }
>
  <div
    style={
      Object {
        "margin": "0 auto",
        "maxWidth": 960,
        "padding": "1.45rem 1.0875rem",
      }
    }
  >
    <h1
      className="MuiTypography-root MuiTypography-h1"
      style={
        Object {
          "fontSize": 45,
          "margin": 0,
          "textAlign": "center",
        }
      }
    >
      <a
        href="/"
        style={
          Object {
            "color": "white",
            "textDecoration": "none",
          }
        }
      >
        Test site title
      </a>
    </h1>
  </div>
</header>
`;

exports[`AppHeader renders without site title when the siteTitle prop is undefined 1`] = `
<header
  style={
    Object {
      "background": "rebeccapurple",
      "marginBottom": "1.45rem",
    }
  }
>
  <div
    style={
      Object {
        "margin": "0 auto",
        "maxWidth": 960,
        "padding": "1.45rem 1.0875rem",
      }
    }
  >
    <h1
      className="MuiTypography-root MuiTypography-h1"
      style={
        Object {
          "fontSize": 45,
          "margin": 0,
          "textAlign": "center",
        }
      }
    >
      <a
        href="/"
        style={
          Object {
            "color": "white",
            "textDecoration": "none",
          }
        }
      >
        
      </a>
    </h1>
  </div>
</header>
`;
